docs/reference/pip_install.rst

```diff
@@ -136,23 +128,38 @@ Lastly, if you wish, you can refer to other requirements files, like this::
 Requirement Specifiers
 ++++++++++++++++++++++
 
-pip supports installing from "requirement specifiers" as implemented in
-`pkg_resources Requirements <http://packages.python.org/setuptools/pkg_resources.html#requirement-objects>`_
+pip supports installing from a package index using a :term:`requirement
+specifier <pypug:Requirement Specifier>`. Generally speaking, a requirement
+specifier is composed of a project name followed by an optional :term:`version
+specifier <pypug:Version Specifier>`.  :ref:`PEP440 <pypa:PEP440s>` contains a
+`full specification
+<https://www.python.org/dev/peps/pep-0440/#version-specifiers>`_ of the
+currently supported specifiers.
 
-Some Examples:
+Below are some examples:
```

Comment:
Please leave this as 'some examples' - below would be appropriate if there was some content between the statement and the list, but the list is adjacent.